libnmstate/nm/bond.py

```diff
@@ -65,7 +65,11 @@ def get_bond_info(nm_device):
     slaves = get_slaves(nm_device)
     options = _get_options(nm_device)
     if slaves or options:
-        return {"slaves": slaves, "options": options}
+        if "mode" in options:
+            # NM.Client might not have bond mode information yet
+            return {"slaves": slaves, "options": options}
+        else:
+            return {}
```

Comment:
I think this is not related to this PR. Could you split this change in another PR? Thanks.

Reply:
Removed. Seems not a problem any more.